### PR TITLE
Added error message under BAD REQUEST responses

### LINKED ISSUE #606: Error message mismatch under update user profile.
When we enter numericals instead of characters for name, the error message mismatches.
 
**To Reproduce**

1. Go to 'user-update user profile'.
2. Provide authorization access token.
3. {
  "name": "1234567",
  "username": "1234567",
  "bio": "string",
  "occupation": "string",
  "organization": "string",
  "slack_username": "string",
  "social_media_links": "string",
  "skills": "string",
  "interests": "string",
  "resume_url": "string",
  "photo_url": "string",
  "need_mentoring": false,
  "available_to_mentor": false,
  "location": "string"
}

4. Click on 'Execute'.
5. Scroll down to 'Response'.
6. 
Screenshot (83)
 

**Expected behavior**
Error message should be same in both server response and responses.

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser  Chrome

### PR DESCRIPTION
**Pull request by faznan3nazer**
### Description
When we enter numericals instead of characters for name, while updating user profile it should show a proper error message and should be listed under responses.

Fixes #606 

### Type of Change:

- Code

### How Has This Been Tested?

Screenshot (367)

### Checklist:

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials

### CODE REVIEW

File: app/api/resources/user.py [hunk 1]
```diff
@@ -129,7 +129,13 @@ def get(cls):
     @users_ns.doc("update_user_profile")
     @users_ns.expect(auth_header_parser, update_user_request_body_model)
     @users_ns.response(HTTPStatus.OK, "%s" % messages.USER_SUCCESSFULLY_UPDATED)
-    @users_ns.response(HTTPStatus.BAD_REQUEST, "Invalid input.")
+    @users_ns.response(
+        HTTPStatus.BAD_REQUEST, 
+        "%s"
+    % (
+        messages.NAME_INPUT_BY_USER_IS_INVALID,
+        ),
```
Comment (devkapilbansal) [inline app/api/resources/user.py]: As there is only one message, this can be written as:
(messages.NAME_INPUT_BY_USER_IS_INVALID,),

**Comment by faznan3nazer** [comment]
@KapilBansal320   I have changed the code now... Is it okay?

**Review by devkapilbansal — APPROVED** [review]
LGTM 👍

**Review by devkapilbansal — DISMISSED** [review]

File: requirements.txt [hunk 2]
```diff
@@ -18,10 +18,6 @@ click==6.7
 colorama==0.3.9
 coverage==4.5.1
 cryptography==2.8
-docker==3.4.0
-docker-compose==1.21.2
-docker-pycreds==0.3.0
-dockerpty==0.4.1
 docopt==0.6.2
```
Comment (rpattath) [inline requirements.txt]: The above changes are irrelevant to the issue you are fixing. Please remove these changes from this PR.

**Comment by faznan3nazer** [comment]
I am closing this PR, and will reopen today itself..Thank you..

**Pull request closed without merging**